cmd/util/ledger/migrations/staged_contracts_migration_test.go

```diff
@@ -508,7 +508,7 @@ func TestStagedContractsMigration(t *testing.T) {
 					Address: stagingAccountAddress,
 					Name:    "MigrationContractStaging",
 				},
-				"ContractUpdate",
+				"MigrationContractStaging.ContractUpdate",
```

Comment:
Does this also have to be updated/fixed in non-test code?

Reply:
it is updated: https://github.com/onflow/flow-go/pull/5659/files#diff-21e3bd4ff5c152211427ad3ebc982745163a76a94c80071d0f8705374663699bR230